calculator.js

```diff
@@ -6,23 +6,102 @@ const exampleAdditionInput = {
 }
 
 const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+  // console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
 
   // Questions to ask and answer:
-  // What is promptInput?
-  // What data type? What does it hold? What does it represent?
-  // How do we read values from it? What syntax?
-  // How can we use it?
-  // Can we call our existing functions now, inside of this function?
+  // What is promptInput? data provided from user
+  // What data type? What does it hold? What does it represent? strings, key: value pairs
+  // How do we read values from it? What syntax? pull from key:value pair, make into number, evaluate if valid, and compare
+  // How can we use it? either directly or by assiging it to a local variable
+  // Can we call our existing functions now, inside of this function? unclear yet how to do helper methods, will update if necessary after function lessons
+
+  let operation = promptInput.operation;
+  let num1 = Number(promptInput.num1);  
+  let num2  = Number(promptInput.num2);
+
+  if (Number.isNaN(num1)  || Number.isNaN(num2)) {
+    if (Number.isNaN(num1) ) {
+      console.log(`${promptInput.num1} is not a number`);
+    } else {
+      console.log(`${promptInput.num2} is not a number`);
+    }
+  } else {
+    if (operation === '+' || operation === 'add') {
+      console.log (`${num1} + ${num2} = ${(num1 + num2)}`);
+    } else if (operation === '-' || operation === 'subtract') {
+      console.log (`${num1} - ${num2} = ${(num1 - num2)}`);
+    } else if (operation === '*' || operation === 'multiply') {
+      console.log (`${num1} * ${num2} = ${(num1 * num2)}`); 
+    } else if (operation === '/' || operation === 'divide') {
+      if (num2 == 0) {
+        console.log('You cannot divide by zero');
+      }
+      else {
+      console.log (`${num1} / ${num2} = ${(num1 / num2)}`); 
+      }
+    } else {
+      console.log('unsupported operator')
+    }
```

Comment:
this works just fine! :) consider, though, using a `switch` statement. It will shorten up the number of characters used to create these conditionals.